### ISSUE TITLE
Tests are failing on latest changes

### ISSUE THREAD

**Issue by MantasVaitkunas** [PAYONE-GmbH/oxid-6#99]
Tests started failing on June 7th:
```
1) Unit_fcPayOne_Extend_Application_Controllers_fcPayOnePaymentView::test__fcpoProcessValidation_BoniMoment_1
Error: Call to a member function getBasket() on null
2) Unit_fcPayOne_Extend_Application_Controllers_fcPayOnePaymentView::test__fcpoProcessValidation_BoniMoment_2
Error: Call to a member function getBasket() on null
3) Unit_fcPayOne_Extend_Application_Controllers_fcPayOnePaymentView::tests__fcpoProcessValidation_Error
Error: Call to a member function getBasket() on null
4) Unit_fcPayOne_Extend_Application_Controllers_fcPayOnePaymentView::tests__fcpoProcessValidation_Ok
Error: Call to a member function getBasket() on null
5) Unit_fcPayOne_Extend_Application_Controllers_fcPayOnePaymentView::test__fcpoPayolutionSaveRequestedValues_ValidBirthdate
Missing argument 2 for Mock_fcpayonepaymentview_4239915e::_fcpoSaveBirthdayData()
6) Unit_fcPayOne_Extend_Application_Controllers_fcPayOnePaymentView::test__fcpoExtractBirthdateFromRequest_Payolution
Missing argument 2 for fcPayOnePaymentView::_fcpoExtractBirthdateFromRequest()
7) Unit_fcPayOne_Extend_Application_Controllers_fcPayOnePaymentView::test__fcpoValidateBirthdayData_Payolution
Missing argument 2 for fcPayOnePaymentView::_fcpoValidateBirthdayData()
```

**Comment by hreinberger** [comment]
thx, we'll have a look!

**Comment by fjbender** [comment]
Fixed with #100
